uniprot-config/src/main/java/org/uniprot/store/config/searchfield/common/SearchFieldConfig.java

```diff
@@ -19,6 +20,10 @@ public interface SearchFieldConfig {
 
     SearchFieldItem getSearchFieldItemByName(String fieldName);
 
+    Optional<SearchFieldItem> findSearchFieldItemByName(String fieldName);
```

Comment:
Unable to find tests for this method. Can you please add tests for found/not found example?